src/Symfony/Component/Notifier/Bridge/Slack/SlackTransport.php

```diff
@@ -21,30 +21,30 @@
 
 /**
  * @author Fabien Potencier <fabien@symfony.com>
+ * @author Daniel Stancu <birkof@birkof.ro>
  *
  * @internal
  *
- * @experimental in 5.0
+ * @experimental in 5.1
```

Comment:
Could you revert this please. 

It is unrelated to this PR.

Reply:
Fixed!